docs_src/microservices/application/Triggers.md

```diff
@@ -1,23 +1,27 @@
-Triggers determine how the app functions pipeline begins execution. The trigger is determined by the `configuration.toml` file located in the `/res` directory under a section called `[Binding]`. Check out the [Configuration Section](#configuration) for more information about the toml file.
+Triggers determine how the app functions pipeline begins execution. The trigger is determined by the `configuration.toml` file located in the `/res` directory under a section called `[Binding]`. Check out the [Configuration Section](../GeneralAppServiceConfig/) for more information about the toml file.
 
-## Message Bus Trigger
+## Edgex Message Bus Trigger
 
-A message bus trigger will execute the pipeline every time data is received off of the configured topic.  
+An EdgeX Message Bus trigger will execute the pipeline every time data is received from the configured Edgex Message Bus `SubscribeTopic`.  The EdgeX Message Bus is the central message bus internal to EdgeX and has a specific message envelope that wraps all data published to this message bus.
+
+There currently are three implementations of the EdgeX Message Bus available to be used. These are `ZeroMQ`, `MQTT` & `Redis Streams`. The implementation type is selected via the `[MessageBus]` configuration described below.
 
 ### Type and Topic configuration 
 Here's an example:
+
 ```toml
-Type="messagebus" 
+[Binding]
+Type="edgex-messagebus" # also can use legacy messagebus for type
 SubscribeTopic="events"
 PublishTopic=""
 ```
-The `Type=` is set to "messagebus". [EdgeX Core Data]() is publishing data to the `events` topic. So to receive data from core data, you can set your `SubscribeTopic=` either to `""` or `"events"`. You may also designate a `PublishTopic=` if you wish to publish data back to the message bus.
-`edgexcontext.Complete([]byte outputData)` - Will send data back to back to the message bus with the topic specified in the `PublishTopic=` property
+The `Type=` is set to `edgex-messagebus` trigger type or the legacy type of `messagebus`. [EdgeX Core Data]() is publishing data to the `events` topic. So to receive data from core data, you can set your `SubscribeTopic=` either to `""` or `"events"`. You may also designate a `PublishTopic=` if you wish to publish data back to the message bus. `edgexcontext.Complete([]byte outputData)` - Will send data back to the message bus on the topic specified by the `PublishTopic=` property
```

Comment:
[EdgeX Core Data]() --> is this supposed to be a link somewhere?

Reply:
Not sure what that was, but just removed the Link markup.

Reply:
Fixed